src/routes/+layout.svelte

```diff
@@ -1,9 +1,11 @@
 <script>
-  import Header from "./Header.svelte";
-  import "./styles.css";
+	import Header from './Header.svelte';
   import { setClient } from "svelte-apollo";
   import apolloClient from "$lib/apollo";
   setClient(apolloClient);
+	import './styles.css';
+	import "../app.css";
```

Comment:
We need to define a standard for formatting and lining code

Reply:
Indeed